README.md

```diff
@@ -116,7 +116,7 @@ from datetime import datetime
 def produce_age(instance):
     return datetime.now().year - instance.birth_year
```

Comment:
You know these 'producers' sure look a lot like what are called 'selectors' in redux: https://redux.js.org/usage/deriving-data-selectors.

Would it be a better term?

Reply:
We agree it wouldnt because Python/django programmers probably not very aware of redux selectors